spec/lib/scholrax/importer/handle_file_reader_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+module Scholrax::Importer
+  RSpec.describe HandleFileReader do
+
+    describe "call" do
+      let(:handle_file_path) { Rails.root.join('spec', 'fixtures', 'dspace_export', 'sample', 'handle') }
+
+      subject { described_class.new(handle_file_path) }
+      it "returns the expected handle" do
+        expect(subject.call).to match("12345/98765")
```

Comment:
@ad96 Using 'match' here isn't wrong but I tend to use 'eq' when dealing with strings (as is the case here) and 'match' only when dealing with regular expressions.